test/Bugs/symbol_tostring.js

```diff
@@ -0,0 +1,20 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+var k;
+function test0() {
+  try {
+    // Below will throw an error and we will try to do implicit toString call on Symbol which will also throw.
+    // Under disableImplicit call we were returning nullptr which was wrong.
+    [] = ab[Object(Symbol())] = null;
```

Comment:
This doesn't seem to be true per https://tc39.github.io/ecma262/#sec-topropertykey / https://tc39.github.io/ecma262/#sec-symbol.prototype-@@toprimitive

This will just put a property with name `Symbol()` (type Symbol) in the object. This code should probably still throw, but not for the same reason